bin/custom/concerttours/web/src/concerttours/controller/constant/ControllerConstant.java

```diff
@@ -0,0 +1,25 @@
+package concerttours.controller.constant;
+
+public interface ControllerConstant {
```

Comment:
why do you need interfaces here, when you can create constant class? public final ControllerConstants
+ Constant -> Constants